src/ensembl/src/shared/components/layout/DocLayout.scss

```diff
@@ -0,0 +1,56 @@
+@import 'src/styles/common';
+
+$sidebarContentWidth: 320px;
+
+.DocLayout {
+  height: 100%;
+  min-height: 550px; // based on the current interface of genome browser
+  overflow: hidden;
+}
+
+.mainWrapper {
+  height: 100%;
+  position: relative;
+}
+
+.main {
+  height: 100%;
+  margin-left: $sidebarContentWidth;
+  margin-right: 48px;
+}
+
+.sidebarWrapper {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  z-index: 1;
+}
+
+.sidebar {
+  flex: 0 0 auto;
+  height: 100%;
+  width: $sidebarContentWidth;
+  background-color: white;
+  padding: 0px 36px;
+
+  .runningHeader {
+    font-size: 12px;
+    font-weight: bold;
+    color: $dark-grey;
+    margin-bottom: 24px;
+  }
+
+  ul {
+    list-style: none;
+    margin-left: -40px;
+    line-height: 48px;
+
+    li.current a {
+      font-weight: bold;
+      color: $dark-grey;
+    }
+  }
+}
```

Comment:
Do we need the whole page to be scrollable? Or is it only the main wrapper that we need to be scrollable? If it is the latter then we can use calc(100vh - 103px) as the height of the main wrapper and get rid of height: 100% in the child elements.

Reply:
The whole page needs to be scrollable, as there may be many entries in the lefthand navigation menu (unless Andrea comes up with something clever further down the line).